strict-stm/strict-stm.cabal

```diff
@@ -0,0 +1,66 @@
+name:                strict-stm
+version:             0.1.0.0
+synopsis:            Type classes for concurrency with STM, ST and timing
+-- description:
+license:             Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+copyright:           2019-2021 Input Output (Hong Kong) Ltd.
```

Comment:
Would it make sense to bump the year already?

Reply:
I thought about it, but decided to bump all of them at once.